ajax/libs/react-datagrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-datagrid",
+  "version": "3.0.0",
+  "filename": "index.js",
+  "description": "React DataGrid",
```

Comment:
Let's make this more descriptive. Not necessary to follow `package.json` when they haven't provided a proper description, is it?

You can use `A carefully crafted DataGrid for React.` instead.